tools/xcodeproj_shims/installers/lldb-settings.sh

```diff
@@ -21,6 +21,13 @@ settings set target.sdk-path $SDKROOT
 settings set target.swift-framework-search-paths $FRAMEWORK_SEARCH_PATHS
 END
 
+if [ ! -z ${BAZEL_LLDB_LOG_FILE+x} ]; then
+  if [ -n $BAZEL_LLDB_LOG_FILE ] && [ -n $BAZEL_LLDB_LOG_FILE_TYPE ]; then
+    mkdir -p $(dirname $BAZEL_LLDB_LOG_FILE)
+    echo "log enable -f $BAZEL_LLDB_LOG_FILE lldb $BAZEL_LLDB_LOG_FILE_TYPE" >> $BAZEL_LLDB_INIT_FILE
```

Comment:
Will the lldb log still be enabled after we removing these attrs? Is it possible to clean it up?

Reply:
How do we turn this off?

Reply:
i thought if user re-generate the project without this attribute, it will be cleaned up?

Reply:
That's correct, re-generating without this will create a `.lldbinit` file without the `log enable` command. Same for other attributes so that's why I followed this pattern